packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -176,7 +176,7 @@ class BlobOnlyStorage implements IDocumentStorageService {
             // some browsers may not populate stack unless exception is thrown
             throw new Error("BlobOnlyStorage not implemented method used");
         } catch (err) {
-            this.logger.sendErrorEvent({ eventName: "BlobOnlyStorageWrongCall" }, err);
+            this.logger.sendTelemetryEvent({ eventName: "BlobOnlyStorageWrongCall" }, err);
```

Comment:
This looks unrelated.

Reply:
And makes sense anyway - code throwing an error should rarely also log that error. Let the caller decide if it's actually a problem worth recording.